src/test/java/uk/nhs/digital/nhsconnect/lab/results/utils/PemFormatterTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.nhs.digital.nhsconnect.lab.results.utils;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PemFormatterTest {
+
+    @Test
+    public void when_certHasExtraWhitespace_then_itIsTrimmed() {
+        String withWhitespace = " -----BEGIN CERTIFICATE-----\n " +
+            "    \t  MIIFXzCCA0egAwIBAgIJALRbCSor9bEbMA0GCSqGSIb3DQEBCwUAMEUxCzAJBgNV \n" +
+            "  \n\n    W/JNIRmhLoeFNGNh8HvhI2PwOCsFiqT1rrCaUtusTyH0Ggs=\n" +
+            "   \r   -----END CERTIFICATE-----";
+        String trimmed = "-----BEGIN CERTIFICATE-----\n" +
+            "MIIFXzCCA0egAwIBAgIJALRbCSor9bEbMA0GCSqGSIb3DQEBCwUAMEUxCzAJBgNV\n" +
+            "W/JNIRmhLoeFNGNh8HvhI2PwOCsFiqT1rrCaUtusTyH0Ggs=\n" +
+            "-----END CERTIFICATE-----";
+        String formatted = PemFormatter.format(withWhitespace);
+        assertThat(formatted).isEqualTo(trimmed);
+        assertThat(formatted).isEqualTo(trimmed);
```

Comment:
```suggestion
        assertThat(formatted).isEqualTo(trimmed);
```
And on the other tests here